irohad/torii/query_service.hpp

```diff
@@ -60,7 +58,7 @@ namespace torii {
     std::shared_ptr<iroha::torii::QueryProcessor> query_processor_;
 
     iroha::cache::Cache<shared_model::crypto::Hash,
-                        std::shared_ptr<shared_model::interface::QueryResponse>,
+                        iroha::protocol::QueryResponse,
```

Comment:
Note, query service is aware about the transport, so keeping shared model object in cache only makes life harder by adding redundant conversions from model to transport objects